build.gradle

```diff
@@ -401,11 +412,15 @@ subprojects {
             // dependencies that break when including them. (context's testCompile
             // depends on core; core's testCompile depends on testing)
             includeTests = false
-            if (project.hasProperty('jmhIncludeSingleClass')) {
+            if (project.hasProperty('jmh.include')) {
```

Comment:
This is renamed to be more in line with [jmh-gradle-plugin#configuration-options](https://github.com/melix/jmh-gradle-plugin#configuration-options).